test/helpers/foundry/foundryup.ts

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env -S node --require "./node_modules/tsx/dist/preflight.cjs" --import "./node_modules/tsx/dist/loader.mjs"
```

Comment:
:information_source:  changes in this file are organizing code in functions and lint